cdap-master/src/main/java/co/cask/cdap/master/startup/HBaseCheck.java

```diff
@@ -90,7 +90,10 @@ public void run() {
                                                       "admin privileges in HBase. Global admin privileges for cdap " +
                                                       "is required for transaction pruning. " +
                                                       "Either disable transaction pruning or grant global admin " +
-                                                      "privilege to cdap in HBase.");
+                                                      "privilege to cdap in HBase or at your own risk override this " +
```

Comment:
perhaps "at your own risk" is a little harsh without explaining the risk. Let's just say, You can override this check by setting....